lib/server/routes/buckets.js

```diff
@@ -1327,18 +1339,98 @@ BucketsRouter.prototype.getFileInfo = function(req, res, next) {
 BucketsRouter.prototype._definitions = function() {
   /* jshint maxlen: 120 */
   return [
-    ['GET', '/buckets', this._verify, this.getBuckets],
-    ['GET', '/buckets/:id', this._validate, this._verify, this.getBucketById],
-    ['POST', '/buckets', this._verify, this.createBucket],
-    ['DELETE', '/buckets/:id', this._validate, this._verify, this.destroyBucketById],
-    ['PATCH', '/buckets/:id', this._validate, this._verify, this.updateBucketById],
-    ['POST', '/buckets/:id/tokens', this._validate, this.createBucketToken],
-    ['GET', '/buckets/:id/files', this._validate, this._verify, this.listFilesInBucket],
-    ['GET', '/buckets/:id/files/:file', this._validate, this._usetoken, this.getFile],
-    ['DELETE', '/buckets/:id/files/:file', this._validate, this._verify, this.removeFile],
-    ['GET', '/buckets/:id/files/:file/info', this._validate, this.getFileInfo],
-    ['POST', '/buckets/:id/files', this._validate, this._verify, this.createEntryFromFrame],
-    ['GET', '/buckets/:id/files/:file/mirrors', this._validate, this._verify, this.listMirrorsForFile]
+    [
```

Comment:
nit: I've contemplated putting these all on new lines as well, but I do think it's more readable to break line 80 character column limit for this case.